index.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300&display=swap');
+
+* {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+      font-family: 'Poppins', sans-serif;
+}
+
+:root {
+      --primary-color: #00B4D8;
+      --secondary-color: #90E0EF;
+      --navyblue-color: #03045E;
+      --white-color: #FFFFFF;
+}
+
+.brand-container {
+      width: 100%;
+      height: 100%;
+}
+
+/* Navigation Code Start */
+.navigation-cont {
+      /* width: 97vw; */
+      height: 15vh;
+      background-color: var(--primary-color);
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      color: var(--navyblue-color);
+}
+
+.name-of-site {
+      margin-right: 4rem;
+      font-size: 2rem;
+      font-weight: 600;
+}
+
+.searchbar {
+      width: 20rem;
+      height: 2rem;
+      background-color: var(--white-color);
+      border-radius: 1rem;
+      display: flex;
+      align-items: center;
+      padding: 1rem;
+      background-color: var(--secondary-color);
+}
+
+.fa-search {
+      font-size: 1rem;
+      color: var(--navyblue-color);
+}
+
+.search-input {
+      border: none;
+      outline: none;
+      padding-left: 0.4rem;
+      font-size: 1rem;
+      flex: 1;
+      height: 2rem;
+      font-weight: 600;
+      background-color: var(--secondary-color);
+}
+
+.nav-icons-name {
+      display: flex;
+      flex-direction: column;
+      font-weight: 600;
+}
+
+.nav-buttons-cont {
+      display: flex;
+}
+
+.nav-button {
+      padding: 0.6rem;
+      font-size: 1rem;
+}
+
+.nav-button i {
+      font-size: 1.5rem;
+}
+
+.login-btn {
+      background-color: var(--secondary-color);
+      border: none;
+      box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -webkit-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+}
```

Comment:
One suggestion here, you can use a variable for reusability (remember DRY principle).